### PR TITLE
fix(build): correct license metadata

### PR DESCRIPTION
**Pull request by gszr**
ASL-2.0 is not a valid SPDX license identifier -- https://spdx.org/licenses/.

### CODE REVIEW

File: build/package/nfpm.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ maintainer: "Kong Inc. <support@konghq.com>"
 description: |
   Kong is a distributed gateway for APIs and Microservices, focused on high performance and reliability.
 vendor: "Kong Inc."
-license: "ASL 2.0"
+license: "Apache-2.0"
```
Comment (hbagdi) [inline build/package/nfpm.yaml:14]: grep reveals:

```
.requirements:KONG_LICENSE="ASL 2.0"
build/package/nfpm.yaml:license: "ASL 2.0"
```

Any reason why KONG_LICENSE was not also included as well?

Comment (gszr) [reply build/package/nfpm.yaml:14]: Good catch.

**Comment by gszr** [comment]
> My approvals are generally not authoratative since I'm not a Gateway Moderator. I'm not sure if you should consider approval as good for merge or not.

Thank you for reviewing, Harry!

I will wait for a final review from @fffonion.

**Comment by fffonion** [comment]
please see my comment at the EE sister PR to adjust them in sync. we need a template over license field as both ce and ee are using the same yaml file to package

**Comment by fffonion** [comment]
discussed with @gszr offline, let's keep this as is and open follow up PR to properly address my comment above

**Comment by tyler-ball** [comment]
@gszr Pinging as a reminder to backport this to relevant branches

**Pull request merged**